src/Libraries/CoreNodesUI/Input/SliderBase.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace DSCoreNodesUI.Input
+{
+    public abstract class SliderBase<T> : BasicInteractive<T>
+    {
```

Comment:
I believe I tried that setting the constraint to where IComparable, and it didn't work. I'll reinvestigate.